src/github.rs

```diff
@@ -843,9 +856,18 @@ impl Repository {
                 include_labels.join(",")
             )))
             .chain(std::iter::once("filter=all".to_owned()))
-            .chain(std::iter::once(format!("sort=created")))
-            .chain(std::iter::once(format!("direction=asc")))
-            .chain(std::iter::once(format!("per_page=100")))
+            .chain(std::iter::once(format!(
+                "sort={}",
+                ordering.get("sort").unwrap_or(&"created")
+            )))
+            .chain(std::iter::once(format!(
+                "direction={}",
+                ordering.get("direction").unwrap_or(&"asc")
+            )))
+            .chain(std::iter::once(format!(
+                "per_page={}",
+                ordering.get("per_page").unwrap_or(&"100")
+            )))
```

Comment:
Maybe it's easier to just allow this is `filters` and only add these if they aren't there?

Reply:
@jackh726 what you're saying would be fine here in the `pulls` endpoint but won't work in the `search` endpoint https://github.com/rust-lang/triagebot/pull/1461/files#diff-ef5fd0828e2727ce9e100f72dc4f18f0f126e393d35108269a9bea5e8a104d5cR907 where `filters` go in the `q` part and the sorting stuff goes in separate query params. Github API is kind of a mess :). Also note that [here](https://github.com/rust-lang/triagebot/pull/1461/files#diff-ef5fd0828e2727ce9e100f72dc4f18f0f126e393d35108269a9bea5e8a104d5cR864) I'm using direction and [here](https://github.com/rust-lang/triagebot/pull/1461/files#diff-ef5fd0828e2727ce9e100f72dc4f18f0f126e393d35108269a9bea5e8a104d5cR907) I'm using order for the same thing but I'm providing an API inside `triagebot` which the only accepted key is `direction`, this is why https://github.com/rust-lang/triagebot/pull/1461/files#diff-ef5fd0828e2727ce9e100f72dc4f18f0f126e393d35108269a9bea5e8a104d5cR911 I check for direction only.